backend/src/packages/articles/libs/helpers/get-detect-article-genre-completion-config.helper.ts

```diff
@@ -4,7 +4,7 @@ import { UNKNOWN_GENRE_KEY } from '~/packages/genres/genre.js';
 const getDetectArticleGenreCompletionConfig = (
   text: string,
 ): CompletionConfig => ({
-  prompt: `You will be provided with a html markup of an article, and your task is to extract a list of possible genres of that article. Provide your output in json format as an array of objects with keys name and key, where name is a readble name of the genre and key - a short snake cased representation of genre name. Place the most relevant genre at the 0 index. In case you can't indentify the genre put an object { key: "${UNKNOWN_GENRE_KEY}", name: "Unknown" } as the first item in resulting array. Here is the text: ${text}.`,
+  prompt: `You will be provided with a html markup of an article, and your task is to extract a list of possible genres of that article. Provide your output in json format as an array of objects with keys name and key, where name is a readble name of the genre and key - a short snake cased representation of genre name. Place the most relevant genre at the 0 index. In case you can't indentify the genre put an object { key: "${UNKNOWN_GENRE_KEY}", name: "Unknown" } as the last item in resulting array. Here is the text: ${text}.`,
```

Comment:
When I was testing the prompt it was working kinda fine, but as it was discussed in Slack, for some reason currently ai misunderstands this part `In case you can't identify the genre put an object { key: "${UNKNOWN_GENRE_KEY}", name: "Unknown" } as the last item in resulting array`. Maybe we should remove it at all? 🤔

Reply:
I tried to delete it but then chatgpt started to suggest actual genres like "Fantasy" when I send random nonsense like "fgsaasasadad" to it. So I decided to keep "unknown" option as last resort so to say.  And when I tested it worked fine ( but you never can be totally sure with chatgpt looks like )